qiskit_aqua/utils/backend_utils.py

```diff
@@ -223,6 +228,6 @@ def _get_ibmq_provider():
     try:
         providers['qiskit.IBMQ'] = get_backends_from_provider('qiskit.IBMQ')
     except Exception as e:
-        logger.debug("Failed to register with Qiskit IBMQ: {}".format(str(e)))
+        logger.debug("Failed to access IBMQ: {}".format(str(e)))
```

Comment:
do you think we should set the logger to `warning` for this one?

Reply:
Sure, I will change to warning